facade/finalizesearch.rst

```diff
@@ -31,6 +31,11 @@ the order of inclusion, and adds to the services. For background information, se
         }
     }
 
+.. note::
+  As of Firely Server version 4.4.0, your Facade should not have an order greater than or equal to 211. 
+  The reason for this is that the MemoryAdministrationConfiguration (order: 211) used by the admin endpoint
```

Comment:
Most people probably don't use the `MemoryAdministrationConfiguration`, so I would put it a bit more generic like: The reason for this is that upon configuring the administration database, Firely Server checks whether an ``ISearchRepository`` is registered. The earliest of these configurations is at order 211.

Reply:
Thanks. 👍